README.md

```diff
@@ -27,36 +24,44 @@ automatically scaling the capacity with the load.
 
 # Overview
 
-This plugin uses [EC2 Spot Fleet](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-fleet.html) or [Auto Scaling Group](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html) to launch instances instead of directly launching them by itself. 
-Both maintain your fleet's target capacity as Spot prices change to maintain
-the fleet within the specified price range. For more information, see 
-[How Spot Fleet Works](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-fleet.html).
+The EC2 Fleet Plugin scales your [Auto Scaling Group](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html) or [EC2 Spot Fleet](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-fleet.html)
+automatically for your Jenkins workload. It handles launching new instances that match the criteria set in your ASG or Spot Fleet e.g. [allocation strategy](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-mixed-instances-groups.html#allocation-strategies), 
+and terminating idle instances that breach that criteria or those in your Jenkins Cloud configuration.
+
+> [!WARNING]  
+> AWS strongly discourages using the SpotFleets because it is now categorized as legacy API with no planned investment. 
+Use Auto Scaling Groups instead.
 
 # Features
 
-- Support EC2 Spot Fleet or Auto Scaling Group as Jenkins Workers
-- Supports all features provided by EC2 Spot Fleet or Auto Scaling Groups
+- Supports EC2 Spot Fleet or Auto Scaling Group as Jenkins Workers
+- Supports all features provided by EC2 Spot Fleet or Auto Scaling Groups e.g. [multiple instance types across Spot and On-Demand](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-mixed-instances-groups.html#allocation-strategies)
 - Auto resubmit failed jobs caused by Spot interruptions
 - No delay scale up strategy: enable ```No Delay Provision Strategy``` in configuration
 - Add tags to EC2 instances used by plugin, for easy search, tag format ```ec2-fleet-plugin:cloud-name=<MyCloud>```
 - Allow custom EC2 API endpoint
 - Auto Fleet creation based on Job label ([details](docs/LABEL-BASED-CONFIGURATION.md))
+- Set a maximum total uses to terminate nodes after running the set number of jobs
+- Set a minimum spare size to keep nodes ready for incoming jobs, even when idle
+- Default unique cloud names for UI users (shown as a suggestion in the form) and JCasC users (`name: ""` will signal the plugin to generate a default name)
```

Comment:
This is only true for UI on Jenkins before version 2.403. I'm not sure how we would want to phrase this though. Can ignore if we just want to focus on the minimum supported version.

Reply:
I will make a note of this so we can udpate it when we make 2.403 the min version.